src/cljs/main/broadfcui/common/input.cljs

```diff
@@ -8,7 +8,7 @@
 (react/defc TextField
   {:get-text
```

Comment:
bikeshedding since dev is down... it might make sense propagate the `trimmed` part up into `Textfield`'s public API. I don't want to waste any of the learning we had last night